src/App.js

```diff
@@ -0,0 +1,42 @@
+// App.js
+import React from 'react';
+import MyComponent from './components/MyComponent';
+import './App.css';
+
+const App = () => {
+  return (
+    <div class="rectangle-container">
+      <input style={{ display: 'none' }} type="file" accept="image/*" />
+      <br />
+      <h2 class="title">Uploading your image</h2>
+      <br />
+      <p class="subtitle">File should be Jpeg, Png,...</p>
+      <div  className="square-container" type="file" accept="image/*" />
```

Comment:
The `div` tag is not self-closing; that's probably why you have to over use `<br />`.